src/enums/filingNames.ts

```diff
@@ -1,5 +1,5 @@
 export enum FilingNames {
-  ADMIN_FREEZE = 'Administer Freeze / Unfreeze',
+  ADMIN_FREEZE = 'Freeze Business',
```

Comment:
Is this filing name used anywhere else where it should be "Unfreeze Business"?

Reply:
Could be, we should probably hold off on merging this until Freeze and Unfreeze is working correctly again in DEV (previous to this code merge)

Reply:
Then please change this PR to a draft and/or add the Do Not Merge label on it.

Reply:
Added, doesn't look like it's used in any other places